auth/management/commands/purge_tokens.py

```diff
@@ -0,0 +1,21 @@
+import sys
+from datetime import datetime, timezone
+
+from django.core.management.base import BaseCommand
+
+from auth.models import EmailToken
+
+
+def fatal(msg):
+    print(msg)
+    sys.exit(1)
+
+
+class Command(BaseCommand):
+    help = "Supprime les demandes de changements d'emails non utilisés de plus de 24h"
+
+    def handle(self, *args, **options):
+        try:
+            EmailToken.objects.filter(expire_at__lt=datetime.now(timezone.utc)).delete()
+        except RuntimeError as err:
+            fatal(err)
```

Comment:
You can remove the try/catch here, as there's no chances a `RuntimeError` is raised for that call.

Also, makes me think that we should add a test [calling the command](https://www.kite.com/python/examples/5566/django-call-a-management-command-programmatically) over an expired entry in the database and assert that it's been properly deleted.